docs/src/test-snapshots-js.md

```diff
@@ -125,3 +129,70 @@ test('example test', async ({ page }) => {
 ```
 
 Snapshots are stored next to the test file, in a separate directory. For example, `my.spec.ts` file will produce and store snapshots in the `my.spec.ts-snapshots` directory. You should commit this directory to your version control (e.g. `git`), and review any changes to it.
+
+
+## Snapshot Suffix Overrides
```

Comment:
I couldn't get the docs to work locally (seem to be missing a docusaurus config?), so I have no clue if all the syntax in the docs is correct and working as intended. 😄 

We're using Plawyright Test more and more for API testing. It seems like others (based on some Slack convos) are also interested in this and/or doing it already. In the future, a dedicated preset-config and/or docs page with tips-and-tricks (which go beyond this setting) may be nice. Happy to contribute it once I have a better sense of what is and isn't working for us in the way we build our API tests.